src/main/java/frc/trigon/robot/subsystems/arm/simulationarm/SimulationArmIOConstants.java

```diff
@@ -0,0 +1,83 @@
+package frc.trigon.robot.subsystems.arm.simulationarm;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.ElevatorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+public class SimulationArmIOConstants {
+    private static final int
+            ANGLE_MOTOR_AMOUNT = 2,
+            ELEVATOR_MOTOR_AMOUNT = 2;
+    private static final DCMotor
+            ANGLE_MOTOR_GEARBOX = DCMotor.getNEO(ANGLE_MOTOR_AMOUNT),
+            ELEVATOR_MOTOR_GEARBOX = DCMotor.getNEO(ELEVATOR_MOTOR_AMOUNT);
+    private static final double
+            ANGLE_GEAR_RATIO = 12.8;
+    private static final double MAX_ELEVATOR_HEIGHT_METERS = 1;
+    private static final Rotation2d
+            MIN_ANGLE_RADIANS = Rotation2d.fromDegrees(5),
+            MAX_ANGLE_RADIANS = Rotation2d.fromDegrees(5);
+    private static final boolean
+            ANGLE_SIMULATE_GRAVITY = true,
+            ELEVATOR_SIMULATE_GRAVITY = true;
+    private static final double RETRACTED_ARM_LENGTH_METERS = 1;
+    private static final double
+            ARM_MASS = 1,
+            ELEVATOR_MASS = 1,
+            DRUM_RADIUS_METERS = 1;
```

Comment:
don't see how this is similar to mass. this should have `private static final double` before it